src/bundle/Resources/views/themes/admin/ui/dashboard/block/ez.html.twig

```diff
@@ -13,103 +13,141 @@
+                    <use xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#information"></use>
+                </svg>
+                <span>
+                    {{ 'dashboard.ez_version.trial_notice'|trans({'%name%': ez.name, '%doc_url%': urls['doc'], '%consulting_url%': urls['consulting_service'], '%training_url%': urls['training_service']})
+                    |desc('Welcome to %name%, check our <a target=\"_blank\" href=\"%doc_url%\">online documentation</a>, <a target=\"_blank\" href=\"%consulting_url%\">consulting</a>
+                        or <a target=\"_blank\" href=\"%training_url%\">training</a> services in order to get the most out of your trial.')
+                    |raw }}
         {% endif %}
-            {{ 'dashboard.ez_version.trial_contact'|trans({'%contact_url%': urls['contact'], '%install_ee%': urls['install_ee']})
-            |desc("<a target=\"_blank\" href=\"%contact_url%\">Contact eZ or its partner(s)</a> to purchase a subscription
-                and follow the <a target=\"_blank\" href=\"%install_ee%\">online documentation</a> to configure composer.json 'repositories' url for 'bul' instead of 'ttl'.")
-            |raw }}
-        </div>
+                    {{ 'dashboard.ez_version.trial_contact'|trans({'%contact_url%': urls['contact'], '%install_ee%': urls['install_ee']})
+                    |desc("<a target=\"_blank\" href=\"%contact_url%\">Contact eZ or its partner(s)</a> to purchase a subscription
+                        and follow the <a target=\"_blank\" href=\"%install_ee%\">online documentation</a> to configure composer.json 'repositories' url for 'bul' instead of 'ttl'.")
+                    |raw }}
+                </span>
+            </div>
     {% elseif not ez.isEnterpise %}
         {% set badge = 'GPL' %}
         {% if ez.isEndOfMaintenance %}
             {# In the future with retrival of info from updates.ez.no we can detect missing (public) security fixes and then let this become an error #}
             {% set severity = 1 %}
-            <div class="alert alert-warning mb-0 mt-3" role="alert">
-                {{ 'dashboard.ez_version.community_end_of_maintenance'|trans({
-                    '%release%': ez.release,
-                    '%update_url%': urls['update']
-                })
-                |desc("Unfortunately %release% open source version has reached end of life,
-                    <a target=\"_blank\" href=\"%update_url%\">please upgrade</a>.")
-                |raw }}
-                <em>
-                    {{ 'dashboard.ez_version.community_end_of_maintenance_upgrade'|trans({
-                        '%license_url%': urls['license'],
-                        '%ee_product_url%': urls['ee_product'],
-                        '%support_service_url%': urls['support_service'],
-                        '%service_life_url%': urls['service_life']
+            <div class="alert alert-warning mb-0 mt-3 ez-alert ez-alert--icon" role="alert">
```

Comment:
Shouldn't here be extra ident? At the moment it's at the same level as `{% if %}`

Reply:
I think it is not at the same level as` if`